packages/storybook/stories/ChatComposite/CustomBehaviorExampleContainer.tsx

```diff
@@ -55,7 +55,12 @@ export const ContosoChatContainer = (props: ContainerProps): JSX.Element => {
   return (
     <div style={{ height: '100vh', width: '100vw' }}>
       {adapter ? (
-        <ChatComposite fluentTheme={props.fluentTheme} locale={props.locale} adapter={adapter} />
+        <ChatComposite
+          fluentTheme={props.fluentTheme}
+          locale={props.locale}
+          adapter={adapter}
+          options={{ participantPane: true }}
```

Comment:
This presumably can be in the PR that sets up CI for stable build, but I think everywhere with the participantPane property needs `/* @conditional-compile-remove-from(stable) */`

Reply:
Yeah, the conditional compilation work isn't complete by any means. But the beta build is unaffected, and is what our CI tests anyway. So deduping first.

Reply:
@PorterNan  ^^

Reply:
@JamesBurnside Actually there is no conditional compilation in storybook. It builds against the beta API, and we call out what features are in preview vs not.

Reply:
Good point!